app/views/instrument_request_approval/new.html.haml

```diff
@@ -0,0 +1,4 @@
+= t_model_error(@instrument.class, :requires_approval, instrument: @instrument)
```

Comment:
We should probably include the current facility as a header on this page. Not sure if that's `content_for` or not.